man/org.freedesktop.systemd1.xml

```diff
@@ -623,8 +623,6 @@ node /org/freedesktop/systemd1 {
 
     <!--signal JobNew is not documented!-->
 
-    <!--signal JobRemoved is not documented!-->
```

Comment:
so, interestingly the JobRemoved signal actually *was* documented properly already, if you scroll down. But the XML scripts didn't grok that. My guess is that the this is some confusion about whether signal names should be suffixed with () or not. Any chance you can clean this up?

@keszybz is this something to fix in the xml scripts?

i#d probably suggest to use () as suffix also for signals.

Reply:
the commit title is a bit misleading of course, given it's already documented in all details

Reply:
> Any chance you can clean this up?

Yeah I noticed as well, but it's a separate issue, so let's do it in another PR so I can publish these changes please

Reply:
Reworded the commit title and added () as suggested

Reply:
filed a bug about the xml script issue so that we don't forget: https://github.com/systemd/systemd/issues/31002